taf/TAF_Metadata.py

```diff
@@ -120,7 +120,7 @@ class TAF_Metadata:
             (493, '012_bsf_ELG00012', '0.1. create_initial_table', 'ELG00012', 'submtg_state_cd', 'submtg_state_cd'),
             (494, '016_bsf_ELG00016', '0.1. create_initial_table', 'ELG00016_{bsf_file_date}_uniq', 'submtg_state_cd', 'submtg_state_cd'),
             (495, '018_bsf_ELG00018', '0.1. create_initial_table', 'ELG00018', 'submtg_state_cd', 'submtg_state_cd'),
-            (498, '023_bsf_ELG00023', '0.1. create_initial_table', 'BSF_{rpt_out}_{bsf_file_date}', 'submtg_state_cd', 'submtg_state_cd'),
+            (498, '023_bsf_ELG00023', '0.1. create_initial_table', 'BSF_STEP1', 'submtg_state_cd', 'submtg_state_cd'),
```

Comment:
It appears after changing this to BSF_STEP1 that there's duplicated lines:
```
            (445, '023_bsf_ELG00023', '0.1. create_initial_table', 'BSF_STEP1', 'distinct msis_ident_num', 'submtg_state_cd'),
            (450, '023_bsf_ELG00023', '0.1. create_initial_table', 'BSF_STEP1', 'submtg_state_cd', 'submtg_state_cd'),
            (477, '023_bsf_ELG00023', '0.1. create_initial_table', 'BSF_STEP1', 'distinct msis_ident_num', 'submtg_state_cd'),
            (498, '023_bsf_ELG00023', '0.1. create_initial_table', 'BSF_STEP1', 'submtg_state_cd', 'submtg_state_cd'),
```
In taf/TAF_Runner.py, `__create_formated_lkup`, the loop goes through all `pgmLkpData` rows, so wouldn't this produce duplicate data?

In the original code there's still a reference to BSF_(rpt_out)_(bsf_file_date):
```
001_batch_bsf.sas: %CREATE_EFTSMETA_INFO(&DA_SCHEMA., &DA_RUN_ID., &TABLE_NAME., 023_bsf_ELG00023, 0.1. create_initial_table, %nrstr(BSF_&RPT_OUT._&BSF_FILE_DATE), submtg_state_cd);
```
Could you explain why this change was made?

Reply:
Good catch! Yes, I should remove the duplicate lines. Talked to @C1KB-tensileai last week when I discovered what the actual issue was and the BSF_STEP1 combined a couple of steps from the SAS code so that makes sense why there is duplicate now. Let me resolve that.

Reply:
Done!